tests/backup/backup_multiple_provisioner_test.go

```diff
@@ -222,3 +222,403 @@ var _ = Describe("{MultipleProvisionerCsiSnapshotDeleteBackupAndRestore}", func(
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{MultipleProvisionerCsiKdmpBackupAndRestore}", func() {
+
+	var (
+		backupNames                               []string
+		cloudCredName                             string
+		cloudCredUID                              string
+		backupLocationUID                         string
+		backupLocationName                        string
+		backupLocationMap                         map[string]string
+		labelSelectors                            map[string]string
+		providers                                 []string
+		schedulePolicyName                        string
+		schedulePolicyUID                         string
+		scheduleUid                               string
+		srcClusterUid                             string
+		schedulePolicyInterval                    = int64(15)
+		scheduledAppContextsForMultipleAppSinleNs []*scheduler.Context
+		scheduledAppContextsForDefaultVscBackup   []*scheduler.Context
+		scheduledAppContextsForCustomVscBackup    []*scheduler.Context
+		allAppContext                             []*scheduler.Context
+		appSpecListDefaultVsc                     []string
+		appSpecListCustomVsc                      []string
+		defaultSchBackupName                      string
+		scheduleList                              []string
+		defaultProvisionerScheduleName            string
+		nonDefaultVscSchBackupName                string
+		randomStringLength                        = 10
+		forceKdmpSchBackupName                    string
+		kdmpScheduleName                          string
+		multipleProvisionerSameNsScheduleName     string
+		multipleNsSchBackupName                   string
+		preRuleNameMultiProvisioner               string
+		postRuleNameMultiProvisioner              string
+		preRuleUidMultiProvisioner                string
+		postRuleUidMultiProvisioner               string
+		appSpecListMultiProvisioner               []string
+		preRuleNameDefaultVsc                     string
+		postRuleNameDefaultVsc                    string
+		preRuleUidDefaultVsc                      string
+		postRuleUidDefaultVsc                     string
+		preRuleNameCustomVsc                      string
+		postRuleNameCustomVsc                     string
+		preRuleUidCustomVsc                       string
+		postRuleUidCustomVsc                      string
+		clusterProviderName                       = GetClusterProvider()
+		provisionerDefaultSnapshotClassMap        = GetProvisionerDefaultSnapshotMap(clusterProviderName)
+		provisionerSnapshotClassMap               = GetProvisionerSnapshotClassesMap(clusterProviderName)
+	)
+
+	JustBeforeEach(func() {
+		if GetClusterProvider() != "ibm" || GetClusterProvider() != "openshift" {
```

Comment:
Can we create a list and check `Contains()`?

Reply:
done